README.md

```diff
@@ -72,6 +72,10 @@ files (by convention, saved in the `spec/policies` directory).
   are permitted by the policy.
 - `permit_edit_and_update_actions` Tests that both the edit and update actions
   are permitted by the policy.
+- `permit_attribute` Tests that an attribute, passed in as a parameter,
+  is permitted by the policy for mass assignment.
+- `permit_attributes` Tests that a list of attributes, passed in as a parameter,
+  are permitted by the policy for mass assignment.
 - `permit_mass_assignment_of(:attribute_name)` or
   `permit_mass_assignment_of(%i[attribute1 attribute2])` Tests that mass
```

Comment:
I think we can change this to simply:

- `permit_mass_assignment_of` Alternative syntax for `permit_attributes`, used for backward compatibility with older test suites.

Same for `forbid_mass_assignment_of`.

That explains the intent of having the additional matcher in a concise sentence.